src/components/Cloud/Alerts-Log.js

```diff
@@ -1,11 +1,13 @@
-import React from 'react';
 import PropTypes from 'prop-types';
+import React from 'react';
+import withApiFetch from '../../utils/withApiFetch';
+import { compose } from 'recompose';
 
 import DashboardBlock from '../Dashboard-Block';
 import AlertItem from './Alert-Item';
 
 /**
- * List all open alerts/activity on a site's server.
+ * List all open alerts/ivity on a site's server.
```

Comment:
search/replace error? not a biggie but maybe it's affected some stuff  unintentionally